app/assets/javascripts/actions/task.js

```diff
@@ -0,0 +1,46 @@
+import actionTypes from './actionTypes';
+
+export const taskAdd = (task, story) => ({
```

Comment:
I think you can change this to `addTaskSuccess`, the same to the actions below. This way you also know that this is the last action called when all goes right.